src/Microsoft.ML.StandardLearners/StandardLearnersCatalog.cs

```diff
@@ -556,5 +556,29 @@ public static PoissonRegression PoissonRegression(this RegressionCatalog.Regress
 
             return new LinearSvmTrainer(CatalogUtils.GetEnvironment(catalog), options);
         }
+
+        /// <summary>
+        /// Predict a target using a random binary classification model <see cref="RandomTrainer"/>.
```

Comment:
><see cref="RandomTrainer"/> [](start = 72, length = 27)

is sentence complete ?

Reply:
I'll try to improve it.

---
In reply to: [256072727](https://github.com/dotnet/machinelearning/pull/2510#discussion_r256072727) [](ancestors = 256072727)